src/main/java/com/avispl/dal/communicator/polycom/videoos/PolycomVideoOS.java

```diff
@@ -0,0 +1,1138 @@
+
+                endpointStatistics.setCallStats(callStats);
+                endpointStatistics.setAudioChannelStats(audioChannelStats);
+                endpointStatistics.setVideoChannelStats(videoChannelStats);
+                endpointStatistics.setContentChannelStats(contentChannelStats);
+            }
+
+            localStatistics = extendedStatistics;
+            localEndpointStatistics = endpointStatistics;
+        } finally {
+            controlOperationsLock.unlock();
+        }
+
+        return Arrays.asList(extendedStatistics, endpointStatistics);
+    }
+
+    /**
+     * Updates the local value of a given controllable property.
+     *
+     * @param controlName name of a controllable property stored in {@link #localStatistics} variable
+     * @param value target value of a controllable property
+     */
+    private void updateLocalControllablePropertyState(String controlName, String value) {
+        if(null != localStatistics) {
+            localStatistics.getControllableProperties().stream().filter(advancedControllableProperty ->
+                    advancedControllableProperty.getName().equals(controlName)).findFirst()
+                    .ifPresent(advancedControllableProperty -> advancedControllableProperty.setValue(value));
+        }
+    }
+
+    /**
+     * Generate CallStatus instance based on callId and callStatusState parameters
+     *
+     * @param callId id of the conference
+     * @param callStatusState state of the call to use
+     * @return {@link CallStatus} instance, indicating the requested status of the call
+     */
+    private CallStatus generateCallStatus(String callId, CallStatus.CallStatusState callStatusState){
+        CallStatus callStatus = new CallStatus();
+        callStatus.setCallId(callId);
+        callStatus.setCallStatusState(callStatusState);
+        return callStatus;
+    }
+
+    /**
+     * Check if outcoming video feed is muted
+     *
+     * @return boolean outcoming video feed mute status
+     * @throws Exception during http communication
+     */
+    private Boolean retrieveVideoMuteStatus() throws Exception {
+        JsonNode response = doGet(VIDEO_MUTE, JsonNode.class);
+        return getJsonProperty(response, "result", Boolean.class);
+    }
+
+    /**
+     * Set local video feed mute status
+     * When operation is succeeded - we need to update {@link #localStatistics} with the new state of the
+     * controllable property.
+     *
+     * @param status boolean indicating the target outcoming video feed state
+     * @throws Exception during http communication
+     */
+    private void updateVideoMuteStatus(boolean status) throws Exception {
+        ObjectNode request = JsonNodeFactory.instance.objectNode();
+        request.put("mute", status);
+        JsonNode response = doPost(VIDEO_MUTE, request, JsonNode.class);
+
+        Boolean success = getJsonProperty(response, "success", Boolean.class);
+        if(Boolean.TRUE.equals(success)) {

```

Comment:
This is weird. So now we are swallowing error? The control operation is asked to be performed, it did not succeed, and there is no error? Why exception is not thrown?